tc/core/polyhedral/schedule_tree.cc

```diff
@@ -343,7 +343,9 @@ vector<const ScheduleTree*> ScheduleTree::collectDFSPreorder(
 vector<const ScheduleTree*> ScheduleTree::collectDFSPreorder(
     const ScheduleTree* tree,
     detail::ScheduleTreeType type) {
-  auto filterType = [type](const ScheduleTree* t) { return t->type_ == type; };
+  auto filterType = [type](const ScheduleTree* t) {
```

Comment:
The commit message is completely decorrelated from the changes which themselves should not have happened.